### PR TITLE
Corrected instruction

### LINKED ISSUE #533: [2-github-basics/] Instruction Misleading
**Describe the bug**
In [github-basic](https://github.com/microsoft/Web-Dev-For-Beginners/tree/main/1-getting-started-lessons/2-github-basics)
Miss `git branch -M main` in Step 10 Connect your local Git repo with GitHub,
and in Step 11 After `git push -u origin master` got following error
image

Another issue is that 
image

"Replace repository_name with your GitHub URL." is misleading.

**To Reproduce**
1. Go to 'https://github.com/microsoft/Web-Dev-For-Beginners/tree/main/1-getting-started-lessons/2-github-basics'
2. Follow the instructions till Step 11 Send local files to GitHub
3. Cannot get the expected result

**Expected behavior**
image

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
The fix should be running  `git branch -M main`  before `git remote add origin ...
`

### PR DESCRIPTION
**Pull request by rauthanshubham1**
Resolved #533 
There was misleading instruction in point 10 and 11. I have resolved it

### CODE REVIEW

File: 1-getting-started-lessons/2-github-basics/README.md [hunk 1]
```diff
@@ -134,14 +134,16 @@ Let's say you have a folder locally with some code project and you want to start
    This creates a _remote_, or connection, named "origin" pointing at the GitHub repository you created earlier.
 
 1. **Send local files to GitHub**. So far you've created a _connection_ between the local repo and the GitHub repo. Let's send these files to GitHub with the following command `git push`, like so: 
+   
+   > Note, replace ```main``` with the branch name you want to push to GitHub.
```
Comment (jlooper) [inline 1-getting-started-lessons/2-github-basics/README.md:138]: We want to encourage folks to use 'main', so let's say:

"Note, your branch name may be different by default from 'main'"

**Review by jlooper — CHANGES_REQUESTED** [review]

File: 1-getting-started-lessons/2-github-basics/README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ Let's say you have a folder locally with some code project and you want to start
 
 1. **Connect your local Git repo with GitHub**. A Git repo is good on your machine but at some point you want to have backup of your files somewhere and also invite other people to work with you on your repo. One such great place to do so is GitHub. Remember we've already created a repo on GitHub so the only thing we need to do is to connect our local Git repo with GitHub. The command `git remote add` will do just that. Type the following command:
```
Comment (wvboy31216) [inline 1-getting-started-lessons/2-github-basics/README.md:126]: git remote add

**Review by wvboy31216 — COMMENTED** [review]
[IntelME.log](https://github.com/microsoft/Web-Dev-For-Beginners/files/7610988/IntelME.log)
[IntelME.log](https://github.com/microsoft/Web-Dev-For-Beginners/files/7610989/IntelME.log)

File: 1-getting-started-lessons/2-github-basics/README.md (hunk 1 above)
Comment (rauthanshubham1) [reply 1-getting-started-lessons/2-github-basics/README.md:138]: Ok, i will update it. Thanks

**Comment by rauthanshubham1** [comment]
@jlooper , Mam changes are done, plz check

**Review by jlooper — COMMENTED** [review]
this is nice...what should we do about the translations?

**Review by wvboy31216 — APPROVED** [review]

**Comment by rauthanshubham1** [comment]
> this is nice...what should we do about the translations?
I know Hindi (Indian Language) so i would try translating that.

**Comment by ManuSquall** [comment]
> this is nice...what should we do about the translations?

Hello world, maybe add a new issue and create labels for all translation langages to make the change?

**Review by jlooper — APPROVED** [review]
accepting, but we need to propagate these changes to the translations. thank you!

**Pull request merged**